### DISCUSSION TITLE
Lab 2 Part 1 Feedback

### DISCUSSION THREAD

**Discussion by shivanidoshi26** [snlee159/Interactive-Lab-Hub#37] [General]
Wow, you guys were super thorough. It was nice that you showed the proof of having done each section by including a video, well done! You guys definitely got a lot accomplished for this lab, which is awesome to see. I like that you have that rotary encoder to shift the time and change the background accordingly. The further interactions part was also really interesting, where you're playing with different hours having different significances. All in all, really great work!

**Comment by anam884** [comment]
GREAT JOB GUYS, 👌👌👌 I thoroughly ENJOYED going over your lab submission and learned a lot too! As someone who's using the Raspberry Pi for the first time makes me really excited about the cool stuff we can do!

**Reply by beibeilu** [reply]
same here
